shared/src/business/useCases/pendingMotion/getPendingMotionDocketEntriesForCurrentJudgeInteractor.ts

```diff
@@ -35,51 +34,52 @@ export const getPendingMotionDocketEntriesForCurrentJudgeInteractor = async (
     throw new UnauthorizedError('Unauthorized');
   }
 
-  const docketNumbers = await getDocketNumbersOfCasesWithStatusType(
-    applicationContext,
-    judge,
-  );
-
-  const allCasesByJudge = await Promise.all(
-    docketNumbers.map(docketNumber =>
-      applicationContext.getPersistenceGateway().getCaseByDocketNumber({
-        applicationContext,
-        docketNumber,
-        includeConsolidatedCases: true,
+  const { results: allDocketEntries } = await applicationContext
+    .getPersistenceGateway()
+    .getAllPendingMotionDocketEntriesForJudge({ applicationContext, judge });
+
+  const currentDate = prepareDateFromString().toISO()!;
+
+  const batchCases = (
```

Comment:
Are these still batchCases?

Reply:
No, I will be refactoring once deployed to get better naming and typing